web/src/components/metrics/metrics-helper.tsx

```diff
@@ -147,6 +147,9 @@ const getPeerName = (
   if (name) {
     return truncateParts(name, truncateLength);
   }
+  if (scope === 'droppedCause' || scope === 'droppedState') {
```

Comment:
Could be for a follow-up PR: looks like these new values aren't really scopes, and since they need sometimes to be excluded, maybe we could compose types such as:

```ts
   type FlowScope = 'host' | ... // (usual flow scopes);
   type AggregateBy = FlowScope | 'droppedCause' | 'droppedState';
```

So we don't mix them too much which will avoid having to handle special cases like that

Reply:
here we go: https://github.com/netobserv/network-observability-console-plugin/pull/324/commits/826ac695d633e5ae975d9e6cfdf6b54615c17663